### ISSUE TITLE
Download the enriched pathways with up and down regulated genes separated

### ISSUE THREAD

**Issue by JianYes** [gexijin/idepGolem#567]
**Is your feature request related to a problem? What value do you see this adding? Please describe.**
For Pathway tab, we need to add download option of the enriched pathways with the normal information such as group, FDR, nGenes, Pathway size, Fold enrichment, Pathway, URL and Genes. It's better to separate the up and down regulated genes in different columns.

**Describe the solution you'd like**
The table may include group, FDR, nGenes, Pathway size, Fold enrichment, Pathway, URL, GenesUp, and GenesDown. 

**Describe alternatives you've considered**
One user suggested to add the columns of number of up regulated genes and down regulated genes. We may express nGenes with two columns as nGenesUp, nGenesDown if it's necessary.

**Additional context**
At least, we need the option to download the enriched pathways with normal information if we don't separate the genes into up and down.

**Comment by ChrisTret** [comment]
Hey Jenny, I just want to make sure I am understanding what is wanted. Here is what the current csv looks like and also a what I imagine we want the "new" csv to look like. Is this what you are picturing. Some of it is just dummy data.
[new.csv](https://github.com/gexijin/idepGolem/files/13747448/new.csv)
[sig_pathways.csv](https://github.com/gexijin/idepGolem/files/13747449/sig_pathways.csv)

**Comment by JianYes** [comment]
Not really, Chris. Right now in these two files we only list the total gene number in the column "Genes" for the enriched pathways. Actually, for each pathway,  we define it as up(or down) when most of the genes included in this pathway are up(or down), not all of the genes are up(or down). The user tried to know how many genes are up and how many are down for each enriched pathway, if it's possible.

**Comment by JianYes** [comment]
For the download csv file for the pathway, at least we need to include the regulated genes! Just like the one in DEG2.

**Comment by gexijin** [comment]
List of genes in pathway analysis results.
